### PR TITLE
clean up log for wq

### LINKED ISSUE #8678: Duplicate records in WorkQueueManager based on WQE
I believe we're somewhere iterating over WQE and logging the message below:
```
2018-07-11 03:55:10,961:139901899704064:INFO:WorkQueueManagerWorkPoller:Pulling work from https://cmsweb.cern.ch/couchdb/workqueue
2018-07-11 03:55:11,050:139901874525952:INFO:WorkQueue:Waiting for parent queue to delete "pdmvserv_task_SUS-RunIIFall17wmLHEGS-00016__v1_T_180415_204129_2221"
2018-07-11 03:55:11,050:139901874525952:INFO:WorkQueue:Waiting for parent queue to delete "pdmvserv_task_SUS-RunIIFall17wmLHEGS-00016__v1_T_180415_204129_2221"
2018-07-11 03:55:11,051:139901874525952:INFO:WorkQueue:Waiting for parent queue to delete "pdmvserv_task_SUS-RunIIFall17wmLHEGS-00016__v1_T_180415_204129_2221"
2018-07-11 03:55:11,051:139901874525952:INFO:WorkQueue:Waiting for parent queue to delete "pdmvserv_task_SUS-RunIIFall17wmLHEGS-00016__v1_T_180415_204129_2221"
```

if not too much work, we should record it only once per workflow name.

### PR DESCRIPTION
**Pull request by ticoann**
fixes #8678

### CODE REVIEW

File: src/python/WMCore/WorkQueue/WorkQueue.py [hunk 1]
```diff
@@ -967,6 +968,10 @@ def performSyncAndCancelAction(self, skipWMBS):
                         self.sendAlert(5, msg='Element for %s stuck for 24 hours.' % wf)
                     for x in updated_elements:
                         self.backend.updateElements(x.id, **x.statusMetrics())
+
+                if not parentQueueDeleted:
+                    self.logger.info('Waiting for parent queue to delete "%s"' % result['RequestName'])
```
Comment (amaltaro) [inline src/python/WMCore/WorkQueue/WorkQueue.py]: replace `result['RequestName']` by `wf`

**Review by amaltaro — CHANGES_REQUESTED** [review]
Thanks, Seangchan. Just a minor detail to fix

**Comment by ticoann** [comment]
Thanks Alan, good catch.

**Comment by ticoann** [comment]
Alan, I updated the patch.

**Review by amaltaro — APPROVED** [review]

**Comment by amaltaro** [comment]
Thanks, it looks good now!

**Comment by ticoann** [comment]
Thanks Alan for the review, merging it.

**Pull request merged**